index.js

```diff
@@ -149,10 +149,10 @@ Discovery.prototype._dhtLookupAndAnnounce = function () {
   self._performedDHTLookup = true
 
   debug('dht lookup')
-  self.dht.lookup(self.infoHash, function (err) {
+  self.dht.lookup(self.infoHashBuffer, function (err) {
```

Comment:
This method can take either a hex string or a buffer, so let's actually pass a hex string since that's the stnadard interface between torrent modules.